app/views/application_templates/show.haml

```diff
@@ -48,4 +48,26 @@
             = r.hidden_field question.unique_prompt_name,
               value: question.prompt
       = hidden_field_tag :position_id, @template.position_id
+      .heading Voluntary Equal Employment Opportunity Information
+      .explanation
+        The University of Massachusetts and the Pioneer Valley
+        Transit Authority are equal opportunity employers. To help
+        us insure that we are complying with EEO policies, please supply
+        the information requested below.
+      .field
+        .label
+          Please select the ethnic group with which you most closely identify
+        = select_tag :ethnicity,
+                     options_for_select(['Prefer not to answer',
+                                         'White (Not of Hispanic origin)',
```

Comment:
Better to have these answers stored in a model constant, since then we can do data validations against them.